### PR TITLE
fix(misc): Fix MarketDetail Asset, Add Prediction Market Disclaimer, Recently Listed sorting

### PR DESCRIPTION
**Pull request by jaredvu**
- Fix `MarketDetail` `AssetIcon`
- Add Prediction Market Disclaimer
- Fix recently listed table
---

## Views

- `<PredictionMarketsIntroDialog>`
  - Add disclaimer

- `<MarketsCompactTable>`
  - Fix sorting for recently listed since clobPairId is pre-assigned in MarketMap

- `<MarketDetails>`
  - pass `logoUrl`

## Constants/Types

- `constants/markets`
  - add prediction-market tag and deprecate old prediction market

## Hooks

- `hooks/useCurrentMarketId`
  - check for prediction market to trigger `PredictionMarkestIntoDialog`

## Packages

- `v4-localization`
  - 
  - updated: v1.1.230 -> v1.1.232

---

### CODE REVIEW

File: src/views/tables/MarketsCompactTable.tsx [hunk 1]
```diff
@@ -146,8 +146,8 @@ export const MarketsCompactTable = ({
   );
 
   const sortedMarkets = useMemo(() => {
-    if (sorting === MarketSorting.HIGHEST_CLOB_PAIR_ID) {
-      return filteredMarkets.sort((a, b) => b.clobPairId - a.clobPairId);
+    if (sorting === MarketSorting.RECENTLY_LISTED) {
+      return filteredMarkets.sort((a, b) => (a.line?.length ?? 0) - (b.line?.length ?? 0));
```
Comment (tyleroooo) [inline src/views/tables/MarketsCompactTable.tsx:150]: huuuuuh what is line and why does it correlate with listing time?

File: src/hooks/useMarketsData.ts [hunk 2]
```diff
@@ -128,6 +132,7 @@ export const useMarketsData = ({
   const sevenDaysSparklineData = usePerpetualMarketSparklines();
   const featureFlags = useAllStatsigGateValues();
   const unlaunchedMarkets = useMetadataService();
+  const hasMarketIds = Object.keys(allPerpetualMarkets).length > 0;
```
Comment (tyleroooo) [inline src/hooks/useMarketsData.ts:135]: this name doesn't feel quite right to me, I'm not quite sure what it means or is used for exactly.

**Review by tyleroooo — APPROVED** [review]

File: src/views/tables/MarketsCompactTable.tsx (hunk 1 above)
Comment (jaredvu) [reply src/views/tables/MarketsCompactTable.tsx:150]: line is the 24H sparkline data. We are sorting by how many line items, so less than 24 means that it has been live for less than a day.

File: src/hooks/useMarketsData.ts (hunk 2 above)
Comment (jaredvu) [reply src/hooks/useMarketsData.ts:135]: Can update, but the gist is that we want to know if the v4_markets channel was successfully subscribed to

**Pull request merged**